experiment/run_experiment.py

```diff
@@ -249,7 +261,8 @@ def start_experiment(  # pylint: disable=too-many-arguments
     validate_experiment_name(experiment_name)
     validate_benchmarks(benchmarks)
 
-    config = read_and_validate_experiment_config(config_filename)
+    config = read_and_validate_experiment_config(config_filename,
+                                                 concurrent_builds)
```

Comment:
I presume this `concurrent_builds` (from the arguments of `start_experiment()`) is the variable we want to configure?

Reply:
We should only get this from the config file. Sorry for not being clear about this.